content/sensu-go/5.12/guides/troubleshooting.md

```diff
@@ -121,13 +122,57 @@ sudo chown -R sensu:sensu /var/cache/sensu/sensu-agent
 
 To troubleshoot handlers and filters, create test events using the [agent API][agent-api], adding the `handlers` attribute to send ad-hoc events to the pipeline.
 
+## Troubleshooting assets {#asset-issues}
+
+Asset filters allow for scoping an asset to a particular operating system or architecture. You can see an example of those in the [asset reference documentation][asset-ref]. If an asset filter is improperly applied, this can prevent the asset from being downloaded to the desired entity, and will result in a message like:
+
+{{< highlight shell >}}
+monitoring-script.sh not found
+{{< /highlight >}}
+
+In the event you see a message like this, it's worth going back and reviewing your asset definition. If you can't remember where you stored the information on disk, you can find it via:
```

Comment:
I think it would also be helpful to show an example from the sensu-agent log illustrating the messages that can be seen when an asset filter doesn't match.